specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/machineLearningServices.json

```diff
@@ -862,6 +862,107 @@
+    "get": {
+      "tags": [
+        "OperationalizationClusters",
+        "MachineLearningComputes",
+        "VirtualMachineSizes"
+      ],
+      "operationId": "Compute_AllowedResizeSizes",
+      "description": "Returns supported virtual machine sizes for resize",
+      "x-ms-long-running-operation": true,
+      "parameters": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+        },
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+        },
+        {
+          "$ref": "#/parameters/WorkspaceNameParameter"
+        },
+        {
+          "$ref": "#/parameters/ComputeNameParameter"
+        },
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+        }
+      ],
+      "responses": {
+        "200": {
+          "description": "OK",
+          "schema": {
+            "$ref": "#/definitions/VirtualMachineSizeListResult"
+          }
+        },
+        "default": {
+          "description": "Error response describing why the operation failed.",
+          "schema": {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+          }
+        }
+      },
+      "x-ms-examples": {
+        "List VM Sizes": {
+          "$ref": "./examples/Compute/allowedVMSizesForResize.json"
+        }
+      }
+    }
+  },
+  "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/computes/{computeName}/resize": {
+    "post": {
+      "tags": [
+        "OperationalizationClusters",
+        "MachineLearningComputes"
+      ],
+      "operationId": "Compute_Resize",
+      "description": "Updates the size of a Compute Instance.",
+      "x-ms-long-running-operation": true,
+      "parameters": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+        },
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+        },
+        {
+          "$ref": "#/parameters/WorkspaceNameParameter"
+        },
+        {
+          "$ref": "#/parameters/ComputeNameParameter"
+        },
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+        },
+        {
+          "name": "parameters",
+          "in": "body",
+          "required": true,
+          "description": "The object for updating VM size setting of specified Compute Instance.",
+          "schema": {
+            "$ref": "#/definitions/ResizeSchema"
+          }
+        }
+      ],
+      "responses": {
```

Comment:
Also include 200 response with the response schema (or return 204 if the response will be empty. Only 202 is not allowed in long running post actions.

Reply:
Fixed

Reply:
This does not make sense for our design